src/graphql/schemas/typeDefs/hyperboardTypeDefs.ts

```diff
@@ -0,0 +1,135 @@
+import { Field, ObjectType } from "type-graphql";
+import { BasicTypeDef } from "./baseTypes/basicTypeDef.js";
+import { EthBigInt } from "../../scalars/ethBigInt.js";
+import { User } from "./userTypeDefs.js";
+import { GraphQLBigInt } from "graphql-scalars";
+
+@ObjectType({
+  description: "Hyperboard of hypercerts for reference and display purposes",
+})
+class Hyperboard extends BasicTypeDef {
+  @Field({ description: "Name of the collection" })
+  name?: string;
+  @Field(() => [EthBigInt], {
+    nullable: true,
+    description: "Chain ID of the hyperboard",
+  })
+  chain_ids?: (bigint | number)[];
+  @Field({ nullable: true, description: "Background image of the hyperboard" })
+  background_image?: string;
+  @Field({
+    nullable: true,
+    description:
+      "Whether the hyperboard should be rendered as a grayscale image",
+  })
+  grayscale_images?: boolean;
+  @Field({
+    nullable: true,
+    description: "Color of the borders of the hyperboard",
+  })
+  tile_border_color?: string;
+
+  @Field(() => [User])
+  admins?: User[];
+
+  @Field(() => SectionWrapper)
+  sections?: SectionWrapper[];
+}
+
+@ObjectType({})
+class SectionWrapper {
```

Comment:
These are generally called something like `SectionResponseType` (`{count, data}`) and overlap with what will be `CollectionResponseType`. So it probably makes sense to follow the naming pattern of the other types. And, looking ahead, it's probably a `CollectionResponseType`. Or would you say Section?

Reply:
I think we discussed calling it a section instead of a collection, to prevent confusion as they do not map 1:1 with the collection entity that will be added to the graph endpoint at a later point. Reason being that a section only can exist within the context of a hyperboard, due to display metadata being linked to a specific hyperboard.

Reply:
I find `Wrapper` confusing, but not a blocker for this PR

Reply:
Yes, that's why I renamed it to response type 👍  This comment still points to an outdated version of the file.